core/src/main/java/org/apache/gravitino/connector/BaseCatalog.java

```diff
@@ -205,10 +205,8 @@ public void initAuthorizationPluginInstance(IsolatedClassLoader classLoader) {
             LOG.info("Authorization provider is not set!");
             return;
           }
-          try {
-            BaseAuthorization<?> authorization =
-                BaseAuthorization.createAuthorization(classLoader, authorizationProvider);
-
+          try (BaseAuthorization<?> authorization =
```

Comment:
Should the authorization be closed after creating plugin? Do we need close method? cc @xunliu

Reply:
I think this change is ok.

Another question is: 
> Should the authorization be closed after creating plugin? Do we need close method? cc @xunliu

We already closed the authorization plugin in the `core/src/main/java/org/apache/gravitino/connector/BaseCatalog.java#L229`

```
  @Override
  public void close() throws IOException {
    if (ops != null) {
      ops.close();
      ops = null;
    }
    if (authorizationPlugin != null) {
      authorizationPlugin.close();
      authorizationPlugin = null;
    }
    if (catalogCredentialManager != null) {
      catalogCredentialManager.close();
      catalogCredentialManager = null;
    }
  }
```

Reply:
In my opinion, `Authorization` class is similar to a factory class. It's weird that the factory class has `close` method.